app/components/e-form-notification.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div :class="b(modifiers)">
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+  import formStates from '@/mixins/form-states';
+
+  export default {
+    name: 'e-form-notification',
+    // components: {},
+    mixins: [formStates],
```

Comment:
- [ ] Please add todo to check mixin documentation after vue-styleguidist update.